scripts/install.sh

```diff
@@ -27,6 +27,25 @@ function success_message {
 }
 trap error_message ERR
 
+function show_help() {
+usage="$(basename "$0") [mode]
+Polycube installation script
+
+mode:
+  pcn-iptables: install pcn-iptables service and related components
```

Comment:
Maybe `only install pcn-iptables` and `only install pcn-k8s` instead of current message.